packages/salesforcedx-vscode-core/src/commands/startApexDebugLogging.ts

```diff
@@ -86,7 +86,7 @@ export const turnOnLogging = async (extensionContext: vscode.ExtensionContext):
   }
 };
 
-const getUserId = async (connection: Connection): Promise<string> => {
+export const getUserId = async (connection: Connection): Promise<string> => {
```

Comment:
nit (non-blocking): I think getUserId belongs on OrgAuthInfo and not inside one of the commands.